public/catalog/v1.0.0/openapi.json

```diff
@@ -1770,7 +1822,17 @@
             }
           }
         }
+      },
+      "BulkQueryIcon": {
+        "type": "object",
+        "properties": {
+          "ids": {
+            "type": "string",
```

Comment:
@lindgrenj6 @abelloti please confirm if this is consistent with the topology approach of doing bulk queries
Not sure but maybe the topology openapi.json might help
https://github.com/ManageIQ/topological_inventory-api/blob/master/public/doc/openapi-3-v0.1.0.json
I would check with @abelloti to be consistent. I know OpenAPI has different ways of implementing it which we are not following in topology I think.

Reply:
@mkanoor it looks like on topology they're using the new filtering feature that was recently implemented. We would have to implement filtering ala https://github.com/ManageIQ/topological_inventory-api/blob/master/app/controllers/application_controller/filter.rb#L105 on our side, which is much more scope than this ticket provides. Closing until we can scope this out.